html-css/week1/index.html

```diff
@@ -0,0 +1,223 @@
+<!DOCTYPE html>
+<html>
+    <head>
+        <meta name="viewport" content="width=device-width, initial-scale=1.0">
+        <title>Responsive CV using Html & CSS</title>
+
+        <link rel="stylesheet" href="https://cdnj.cloudflare.com/ajax/libs/font-awesome/4.7.0/css/font-awesome.css"
+        intergrity="sha512-5A8nwdMOWrSz20fDsjczgUidUBR81iPYU+WymTZP11mY9G6Oc7H1Zv156XqnsgNUzTyMefFcsFH/tnJE/+XBg=="
+        crossorigin="anonymous"/>
+<link rel="stylesheet" type="test/css" href="css/style.css">
+
+    </head>
+    <body>
+
+        <div class="resume-box">
+            <div class="left-section">
+                <div class="profile">
+                    <img src="image/profile.png" class="profile-img">
+                    <div class="blue-box"></div>
+                
+                <h2 class="name"Ngah<span>Anita</span></h2>

```

Comment:
I think you might have forgotten a closing bracket on the `<h2>`
```suggestion
                <h1 class="name">Ngah <span>Anita</span></h1>

```

Also, it's recommended to start the page with the biggest H tag, which would he `<h1>`. You can read more about the order of the headlines here - https://developer.mozilla.org/en-US/docs/Web/HTML/Element/Heading_Elements#usage_notes